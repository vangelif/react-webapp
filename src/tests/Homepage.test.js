import '@testing-library/jest-dom';
import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';

import { configureStore } from '@reduxjs/toolkit';
import Homepage from '../components/Homepage';
import myCountries from './Countries';

const reducer = (
  state = {
    countries: { countries: myCountries },
  }
) => state;

const store = configureStore({ reducer });

describe('Homepage test', () => {
  it('will test if Homepage is rendering correctly', () => {
    const tree = render(
      <MemoryRouter>
        <Provider store={store}>
          <Homepage />
        </Provider>
      </MemoryRouter>
    );
    expect(tree).toMatchSnapshot();
  });
});
