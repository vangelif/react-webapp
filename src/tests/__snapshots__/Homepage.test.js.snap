// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage test will test if Homepage is rendering correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <li
        class="MuiImageListItem-root css-l086p4-MuiImageListItem-root"
        style="height: auto; grid-column-end: span 1; grid-row-end: span 1;"
      >
        <img
          alt="World Map"
          class="MuiImageListItem-img world-icon"
          src="worldmap.jpeg"
          style="width: 100%; height: 100%; object-fit: cover;"
        />
        <div
          class="MuiImageListItemBar-root MuiImageListItemBar-positionBottom css-ztrqiq-MuiImageListItemBar-root"
        >
          <div
            class="MuiImageListItemBar-titleWrap MuiImageListItemBar-titleWrapBottom css-1lizpae-MuiImageListItemBar-titleWrap"
          >
            <div
              class="MuiImageListItemBar-title css-dasnyc-MuiImageListItemBar-title"
            >
              World Map Statistics
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-o0iz4t"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1lujp9z-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="InfoIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </li>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Oceania"
                class="map-icon"
                src="oceania.PNG"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Oceania
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="SouthAmerica"
                class="map-icon"
                src="southamerica.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                South America
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="NorthAmerica"
                class="map-icon"
                src="northamerica.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                North America
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Africa"
                class="map-icon"
                src="africa.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Africa
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Asia"
                class="map-icon"
                src="asia.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Asia
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Antarctic"
                class="map-icon"
                src="antarctic.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Antarctic
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Europe"
                class="map-icon"
                src="europe.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Europe
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <li
      class="MuiImageListItem-root css-l086p4-MuiImageListItem-root"
      style="height: auto; grid-column-end: span 1; grid-row-end: span 1;"
    >
      <img
        alt="World Map"
        class="MuiImageListItem-img world-icon"
        src="worldmap.jpeg"
        style="width: 100%; height: 100%; object-fit: cover;"
      />
      <div
        class="MuiImageListItemBar-root MuiImageListItemBar-positionBottom css-ztrqiq-MuiImageListItemBar-root"
      >
        <div
          class="MuiImageListItemBar-titleWrap MuiImageListItemBar-titleWrapBottom css-1lizpae-MuiImageListItemBar-titleWrap"
        >
          <div
            class="MuiImageListItemBar-title css-dasnyc-MuiImageListItemBar-title"
          >
            World Map Statistics
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-o0iz4t"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1lujp9z-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="InfoIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </li>
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Oceania"
              class="map-icon"
              src="oceania.PNG"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Oceania
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="SouthAmerica"
              class="map-icon"
              src="southamerica.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              South America
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="NorthAmerica"
              class="map-icon"
              src="northamerica.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              North America
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Africa"
              class="map-icon"
              src="africa.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Africa
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Asia"
              class="map-icon"
              src="asia.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Asia
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Antarctic"
              class="map-icon"
              src="antarctic.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Antarctic
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Europe"
              class="map-icon"
              src="europe.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Europe
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`User Interactions test will test user interaction 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <li
        class="MuiImageListItem-root css-l086p4-MuiImageListItem-root"
        style="height: auto; grid-column-end: span 1; grid-row-end: span 1;"
      >
        <img
          alt="World Map"
          class="MuiImageListItem-img world-icon"
          src="worldmap.jpeg"
          style="width: 100%; height: 100%; object-fit: cover;"
        />
        <div
          class="MuiImageListItemBar-root MuiImageListItemBar-positionBottom css-ztrqiq-MuiImageListItemBar-root"
        >
          <div
            class="MuiImageListItemBar-titleWrap MuiImageListItemBar-titleWrapBottom css-1lizpae-MuiImageListItemBar-titleWrap"
          >
            <div
              class="MuiImageListItemBar-title css-dasnyc-MuiImageListItemBar-title"
            >
              World Map Statistics
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-o0iz4t"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1lujp9z-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="InfoIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </li>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Oceania"
                class="map-icon"
                src="oceania.PNG"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Oceania
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="SouthAmerica"
                class="map-icon"
                src="southamerica.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                South America
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="NorthAmerica"
                class="map-icon"
                src="northamerica.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                North America
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Africa"
                class="map-icon"
                src="africa.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Africa
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Asia"
                class="map-icon"
                src="asia.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Asia
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Antarctic"
                class="map-icon"
                src="antarctic.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Antarctic
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <img
                alt="Europe"
                class="map-icon"
                src="europe.jpeg"
                style="width: 100%; height: 100%; object-fit: cover;"
              />
              <div
                class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
              >
                Europe
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <li
      class="MuiImageListItem-root css-l086p4-MuiImageListItem-root"
      style="height: auto; grid-column-end: span 1; grid-row-end: span 1;"
    >
      <img
        alt="World Map"
        class="MuiImageListItem-img world-icon"
        src="worldmap.jpeg"
        style="width: 100%; height: 100%; object-fit: cover;"
      />
      <div
        class="MuiImageListItemBar-root MuiImageListItemBar-positionBottom css-ztrqiq-MuiImageListItemBar-root"
      >
        <div
          class="MuiImageListItemBar-titleWrap MuiImageListItemBar-titleWrapBottom css-1lizpae-MuiImageListItemBar-titleWrap"
        >
          <div
            class="MuiImageListItemBar-title css-dasnyc-MuiImageListItemBar-title"
          >
            World Map Statistics
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-o0iz4t"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1lujp9z-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="InfoIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </li>
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Oceania"
              class="map-icon"
              src="oceania.PNG"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Oceania
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="SouthAmerica"
              class="map-icon"
              src="southamerica.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              South America
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="NorthAmerica"
              class="map-icon"
              src="northamerica.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              North America
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Africa"
              class="map-icon"
              src="africa.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Africa
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Asia"
              class="map-icon"
              src="asia.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Asia
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Antarctic"
              class="map-icon"
              src="antarctic.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Antarctic
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-4 css-14a3zbu-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-83tqv2-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <img
              alt="Europe"
              class="map-icon"
              src="europe.jpeg"
              style="width: 100%; height: 100%; object-fit: cover;"
            />
            <div
              class="MuiTypography-root MuiTypography-h6 css-17qq40j-MuiTypography-root"
            >
              Europe
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
